main.js

```diff
@@ -0,0 +1,58 @@
+const bookList = document.querySelector('ul.books');
+const addForm = document.querySelector('form.add-book');
+const storageDetails = localStorage.getItem('books');
+let bookDetails = [];
+
+// Add books function
+function addBook(bookName, author) {
+  const book = {};
+  book.bookName = bookName;
+  book.author = author;
+  bookDetails.push(book);
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea if we use a non-mutating array method such as `concat` over `push`. In this way, we avoid modifying the original array.